docs/index.md

```diff
@@ -37,28 +37,16 @@ The paper is also available from [ACM Digital Library](https://dl.acm.org/doi/10
 Other publications and presentations are
 linked [here](https://theopenroadproject.org/publications/).
 
-## Code of conduct
-
-Please read our code of conduct [here](main/CODE_OF_CONDUCT.md).
-
 ## Documentation
 
 The OpenROAD Project has two releases:
 
-### [Application](https://github.com/The-OpenROAD-Project/OpenROAD)
-
-The application is a standalone binary capable of performing RTL-to-GDSII SoC design,
-from logic synthesis and floorplanning through detailed routing with metal fill insertion,
-signoff parasitic extraction and timing analysis.
+- Application ([github](https://github.com/The-OpenROAD-Project/OpenROAD)) ([docs](main/README.md)): The application is a standalone binary capable of performing RTL-to-GDSII SoC design.
```

Comment:
Change description to : the application is a standalone binary for digital place and route that can be used by any other RTL-GDSII flow controller.